src/ProductConstructionService/ProductConstructionService.Api/Api/v2020_02_20/Controllers/CodeFlowController.cs

```diff
@@ -54,4 +76,86 @@ await _workItemProducerFactory
 
         return Ok();
     }
+
+    [HttpGet("status")]
+    [SwaggerApiResponse(HttpStatusCode.OK, Type = typeof(List<SyncedCommit>), Description = "List of Synced Commits")]
+    public async Task<IActionResult> GetCodeFlowStatus(string branch)
+    {
+        string vmrUrl = "https://github.com/dotnet/dotnet";
```

Comment:
I guess this is fine for now, but it should probably allow azdo vmrs too

Reply:
maybe just having it as a parameter

Reply:
I think we don't need this, we removed the VMR page

Reply:
Ah ok, yeah if we're removing the page we can revert this